AllASPFuncs.py

```diff
@@ -859,42 +888,45 @@ def getWeights(cutout_wcs_list, size, snra, sndec, error=None,
 
 def makeGrid(adaptive_grid, images, size, ra, dec, cutout_wcs_list,
              percentiles=[], single_grid_point=False,
-             make_exact=False, makecontourGrid=False):
+             make_exact=False, contour_grid=False):
     '''
     This is a function that returns the locations for the model grid points
     used to model the background galaxy. There are several different methods
     for building the grid, listed below, and this parent function calls the
     correct function for which type of grid you wish to construct.
 
-    TODO: Each type of grid gets their own function.
-    TODO: is npoints even used any more
     TODO: refactor
 
-
     Inputs:
     adaptive_grid: bool, whether to use the adaptive grid. Adaptive grids use
         some information about the image to inform where the grid points are
         placed. If false, a regular grid is used.
 
-
-    returns:
+    Returns:
     ra_grid, dec_grid: numpy arrays of floats of the ra and dec locations for
                     model grid points.
-
-
     '''
     if adaptive_grid:
+        if contour_grid:
```

Comment:
This is a confusing nesting.  I think you can simplify to make this a one level if/elif/else construct.